src/NetPanzer/Core/main.cpp

```diff
@@ -251,6 +251,8 @@ BaseGameManager* initialise(int argc, char** argv) {
     filesystem::addToSearchPath(dataDir.c_str());
   } else {
     LOGGER.warning("Data directory not specified! Game will only work if binary is in source root.");
+    if (!filesystem::exists("maps"))
```

Comment:
I think the fail data test didn't work because netpanzer wasn't exiting cleanly enough, so I added this.

Reply:
Nope, that didn't fix it...